lib/tether/tether.go

```diff
@@ -251,20 +252,32 @@ func (t *tether) setNetworks() error {
 }
 
 func (t *tether) setMounts() error {
+	// provides a lookup from path to volume reference.
+	NameLookupMap := make(map[string]string, 0)
```

Comment:
@matthewavery Agree with caglar - shouldn't be capitalized.
You can also drop an extra map lookup and skip the `targetRef` indirection unless you must have it for logging:
```go
	mounts := make([]string, len(t.config.Mounts))
	nameLookupMap := make(map[string]*url.URL, len(t.config.Mounts))
	for k, v := range t.config.Mounts {
		mounts = append(mounts, v.Path)
		nameLookupMap[v.Path] = &v.Source
	}
```